plugins/workspace-backpack/README.md

```diff
@@ -74,5 +74,23 @@ open it will be closed.
 pixel units relative to the Blockly injection div.
 - `position`: Positions the backpack UI element.
 
+## Compatibility
```

Comment:
This is a great idea! More plugins probably need this section.

While you're here, I might also mention that you use the registered Flyout classes. I'm curious how compatible this will work out to be with the continuous-toolbox which registers a custom flyout.

Reply:
It looks like it is compatible with the continuous toolbox after testing.
I've added a section about how it uses the registered Flyout classes.